### PR TITLE
feat: compatibility and docs Modeler 5.0.0

### LINKED ISSUE #50: Migrate plugins in this repo to Modeler 5.x
**What to do**

Migrate the plugins in this repo to Camunda Modeler 5.x

- [ ] Migration done, and tested
- [ ] Badge for compability added

**Why to do it**

To embrace Camunda Modeler 5.x architecture.

Also see https://docs.google.com/presentation/d/15tjO2EuBqXlNf6vx6bZtIBWiyTBvHGXdON-0EJEfy7M/edit#slide=id.g112d943d662_1_0

### PR DESCRIPTION
**Pull request by Skaiir**
Related to #56
Closes #50

New PR because VSCode absolutely destroyed my other branch.
Continuation of https://github.com/camunda/camunda-modeler-plugins/pull/57

### CODE REVIEW

File: camunda-transaction-boundaries-plugin/README.md [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
 
-If you're interested in how to create your own plugins see the [documentation](https://github.com/camunda/camunda-modeler/tree/master/docs/plugins) and this [example](https://github.com/camunda/camunda-modeler-plugin-example).
+If you're interested in how to create your own plugins see the [documentation](https://docs.camunda.io/docs/components/modeler/desktop-modeler/plugins/) and this [example](https://github.com/camunda/camunda-modeler-plugin-example).
+
+## Compatibility Notice
+
+| Plugin Version | Compatible With Camunda Modeler |
+|-|-|
+| <= 0.4.2 | 2.2 - 4.x |
+| >= 0.5.0 | 5.0.0+ |
```
Comment (MaxTru) [inline camunda-transaction-boundaries-plugin/README.md:18]: How do we release this plugin? Just updating the version in package.json? If so, would we not need to do this as part of this PR?

File: camunda-transaction-boundaries-plugin/menu/menu.js [hunk 2]
```diff
@@ -4,15 +4,7 @@ module.exports = function(electronApp, menuState) {
   return [{
     label: 'Toggle Transaction Boundaries',
     accelerator: 'CommandOrControl+t',
-    enabled: function() {
```
Comment (MaxTru) [inline camunda-transaction-boundaries-plugin/menu/menu.js:7]: Why this change? I liked the comments. I don't think we have to use arrow functions here

**Review by MaxTru — CHANGES_REQUESTED** [review]
Looks good!

I have 2 small questions

File: camunda-transaction-boundaries-plugin/README.md (hunk 1 above)
Comment (Skaiir) [reply camunda-transaction-boundaries-plugin/README.md:18]: That's a good point, I'll double check but I think you are right.

File: camunda-transaction-boundaries-plugin/menu/menu.js (hunk 2 above)
Comment (Skaiir) [reply camunda-transaction-boundaries-plugin/menu/menu.js:7]: I'm happy to revert, I just favor arrow functions for one liners.

**Review by MaxTru — APPROVED** [review]
